Reference/google/protobuf/any_test.pb.swift

```diff
@@ -66,20 +66,18 @@ struct ProtobufUnittest_TestAny: SwiftProtobuf.Message, SwiftProtobuf._MessageIm
 
     init() {}
 
-    func copy() -> _StorageClass {
-      let clone = _StorageClass()
-      clone._int32Value = _int32Value
-      clone._anyValue = _anyValue
-      clone._repeatedAnyValue = _repeatedAnyValue
-      return clone
+    init(storage source: _StorageClass) {
```

Comment:
I've been noting the trend towards using participle phrases as arguments (`mutating: foo` or `unpacking: bar`) and realized that the trend could apply here as well.  But I don't have a strong opinion either way.  And since this is a private internal method, the argument naming is not a critical concern.